### DISCUSSION TITLE
Does/Can AutoMerge bypass review requirements?

### DISCUSSION THREAD

**Discussion by knechtionscoding** [renovatebot/renovate#18264] [Request Help] [answered]
AutoMerge is configured, and listed as enabled on a PR.

`Automerge: Enabled.`

I don't see it being automerged. The checks have passed, the only thing I can think of is that we have required review on the PR so it can't merge/bypass that. Is this just a setting I missed somewhere? Is it possible to bypass review requirements on PRs that have automerge enabled?

**Comment by rarkins** [comment]
Which platform and method of hosting are you using?

**Reply by knechtionscoding** [reply]
Github, self-hosted renovate.

**Reply by rarkins** [reply, marked as answer]
Please read https://docs.renovatebot.com/key-concepts/automerge/#required-pull-request-reviews (although I recommend the whole doc)
